src/ccache.cpp

```diff
@@ -86,6 +86,8 @@ Common options:
                               (normally not needed as this is done
                               automatically)
     -C, --clear               clear the cache completely (except configuration)
+    -e, --evict-older-than N  delete files older than N seconds (this will not
```

Comment:
How about using days as the unit instead of seconds? That would address @dyadav7's suggestion. I don't see a realistic need for other units. Do you?

Reply:
With smallest unit (seconds) a user could can target any unit as needed. Days feels restrictive. And with smaller unit I can better contain results while repetitively compiling same project using git-bisect.

However I dont mind parsing S(econd)/D(ay) to decide what the arg means.  And default to days.

Reply:
> And with smaller unit I can better contain results while repetitively compiling same project using git-bisect.

I don't understand this use case. Can you tell me a bit more why you feel the need to evict results based on time when compiling the same project repetitively?

Regardless, if you want to introduce units then use proper symbols: "s" for second and "d" for day, i.e. lowercase.

Reply:
The use case is: Keep ccache archive for each commit, and remove the unneeded archives once the target commit is found.

Reply:
OK, thanks for the clarification. But why not let the normal LRU eviction handle this?

Reply:
@sjamgade: Have you thought about my above question ("why not let the normal LRU eviction handle this?")?